NuGet/Realm/Realm.nuspec

```diff
@@ -17,10 +17,10 @@
     <tags>Realm database db persistence sqlite</tags>
     <dependencies>
       <group targetFramework="portable-net45+sl5+wp8+wpa81+win8+monoandroid+Xamarin.iOS10+monotouch+Xamarin.Mac">
-        <dependency id="Realm.Database" version="$version$"/>
+        <dependency id="Realm.Database" version="$version$" exclude="native" />
```

Comment:
I don't think NuGet handles iOS and Android in the runtimes folder. And technically the iOS wrappers are a build-time dependency because they're a static library.